target_postgres/target_tools.py

```diff
@@ -188,3 +188,10 @@ def _run_sql_hook(hook_name, config, target):
         with target.conn.cursor() as cur:
             cur.execute(config[hook_name])
             LOGGER.debug('{} SQL executed'.format(hook_name))
+
+    hook_file = hook_name + '_file'
```

Comment:
Should we add a warning or raise an error if both a string and a file are provided? It might create conflicts and we don't have a way to say if one should take precedence or be run first.

@laurentS wdyt?

Reply:
The text property version in config will always run before the file version so I don't think that will be much of an issue with regard to the ordering.